tests/sys/shell_scripting/Makefile

```diff
@@ -0,0 +1,14 @@
+DEVELHELP=0
```

Comment:
Did this sneak in or is there a reason to force `DEVELHELP=0`?

Reply:
I think I just copied that from somewhere